CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.13.0)
 project(bpftrace)
 
+if(${CMAKE_VERSION} VERSION_EQUAL 3.3.0 OR ${CMAKE_VERSION} VERSION_GREATER 3.3.0)
```

Comment:
We set the minimum required CMake version to 3.13 above, so I don't think this check is needed?

Reply:
Yeah, agreed

Reply:
Ah, of course, I just copied the thing from BCC.